Source/ViewerController.swift

```diff
@@ -428,11 +428,11 @@ extension ViewerController: ViewerItemControllerDelegate {
 }
 
 extension ViewerController: ViewerItemControllerDataSource {
-    func overlayIsHidden() -> Bool {
+    func viewerItemControllerIsOverlayHidden(_ viewerItemController: ViewerItemController) -> Bool {
```

Comment:
This needs better naming or maybe a whole refactoring. I'll start with one of your suggestions/